pkg/gui/recent_repos_panel.go

```diff
@@ -59,6 +59,24 @@ func (gui *Gui) updateRecentRepoList() error {
 	return gui.Config.SaveAppState()
 }
 
+// canShowIsPrivateRepo returns true if a private repo is never opend before in lazygit
+func (gui *Gui) canShowIsPrivateRepo() bool {
```

Comment:
I think this function could have a better name. Maybe something like 'IsNewPrivateRepo'

Reply:
That's indeed a better name thx for the suggestion i'll change that.